index.html

```diff
@@ -129,11 +134,21 @@ <h3>Google Summer of Code</h3>
         </p>
 
     <h3>Outreachy (winter rounds only)</h3>
+<div class="info">
+	<p>Outreachy provides three-month internships for people from groups traditionally underrepresented in tech. Interns are paid a stipend of $5,500 and have a $500 travel stipend available to them. Interns work remotely with mentors from Free and Open Source Software (FOSS) communities on projects ranging from programming, user experience, documentation, illustration and graphical design, to data science.</p>
```

Comment:
@amanvats I see that we are picking the content from the about us region of the specific sites, which is good.
But I feel that the description should be approximately of same length for all programs. It will keep things more aesthetically appealing and uniform.
Also, interested people will anyway go to the site and read the details, what do you think?

Reply:
@Janiceilene what are your thoughts on this.

Reply:
I agree that if we're going to include descriptions, then they should all be similar in length and scope (preferably short).  I really like the eligibility addition, but I wouldn't include the specific stipend amount.  That kind of information can change without us noticing and making the change on our website.